client/hooks/useStorageState.ts

```diff
@@ -39,18 +30,17 @@ export function useStorageState(key: string): UseStateHook<string> {
 
   // Get
   React.useEffect(() => {
-    if (Platform.OS === 'web') {
-      try {
-        if (typeof localStorage !== 'undefined') {
-          setState(localStorage.getItem(key));
-        }
-      } catch (e) {
-        console.error('Local storage is unavailable:', e);
-      }
-    } else {
-      SecureStore.getItemAsync(key).then(value => {
```

Comment:
Why remove secure store for native? I think we should bring this part back, leave expo AsyncStorage for web